app/services/cloud_connector_service.rb

```diff
@@ -44,26 +42,26 @@ def send_to_cloud_controller
 
     # Send the request
     response = http.request(request)
-    Rails.logger.info("Sent message for #{@mqtt_client_guid} #{response.code} #{response.message}")
+    Rails.logger.info("Sent message for #{@cloud_connector_id} #{response.code} #{response.message}")
 
     VALID_STATUS_CODES.include?(response.code) ? @task.update!(:controller_message_id => JSON.parse(response.body)['id']) : task_failed(response.body)
   rescue => error
     task_failed(error)
   end
 
   def task_failed(error)
-    Rails.logger.error("Error sending message to cloud controller node id: #{@mqtt_client_guid} #{error}")
+    Rails.logger.error("Error sending message to cloud controller node id: #{@cloud_connector_id} #{error}")
     @task.update_attributes(:state => "completed", :status => "error", :output => {'errors' => ["#{error}"]} )
   end
 
   def validate_options
-    unless @options[:task_id].present? && @options[:task_url].present? && @options[:mqtt_client_url].present? && @options[:mqtt_client_guid].present?
-      raise("Options must have task_id, task_url, mqtt_client_url and mqtt_client_guid keys")
+    unless @options[:task_id].present? && @options[:task_url].present? && @options[:cloud_connector_url].present? && @options[:cloud_connector_id].present?
+      raise("Options must have task_id, task_url, cloud_connector_url and cloud_connector_id keys")
     end
   end
 
   def publish
```

Comment:
@hsong-rh this method is not used anymore it was for MQTT now we directly do a HTTP post to the cloud controller

Reply:
@hsong-rh I meant the publish method, if you look at that code it is for MQTT

Reply:
Sorry, misunderstood